src/PKSim.Presentation/Presenters/Charts/EditTimeProfileAnalysisChartPresenter.cs

```diff
@@ -183,7 +183,7 @@ private void showAnalysis()
       private void calculatePKAnalysis()
       {
          var chartData = CreateChartData();
-         _pkAnalysisPresenter.CalculatePKAnalysis(PopulationDataCollector, chartData);
+         _pkAnalysisPresenter.CalculatePKAnalyses(PopulationDataCollector, chartData, PopulationAnalysisChart.PopulationAnalysis);
```

Comment:
I have rewritten this completely so that the PKCalculation was done ion pkAnalysisPresenter and not in this presenter.
TELL DON'T ASK.